database-service/Dockerfile

```diff
@@ -5,21 +5,12 @@ COPY config.yaml /app
 
 WORKDIR /app
 
-FROM python:3.8-slim AS virtualenv-image
+FROM python:3.8-slim AS unittest
 
-RUN python -m venv /opt/venv
-ENV PATH="/opt/venv/bin:$PATH"
+COPY --from=app /bin/hasura-cli /usr/local/bin/hasura
 
 COPY tests/requirements.txt .
-RUN pip install --no-cache-dir -r requirements.txt
-
-FROM python:3.8-slim AS unittest
-
-RUN apt-get -y update \
-  && apt-get install -y curl \
-  && curl -L https://github.com/hasura/graphql-engine/raw/master/cli/get.sh | bash
 
-COPY --from=virtualenv-image /opt/venv /opt/venv
+RUN pip install --no-cache-dir -r requirements.txt
 
-ENV PATH="/opt/venv/bin:$PATH"
 WORKDIR /app
```

Comment:
I completely got rid of this workaround as I was not able to run the tests anymore on the `unittest` image, because of missing python modules. I guess the virtual environment needs to be activated somehow in that case, which is annoying. I decided to go back to the initial `Dockerfile` version which is much simpler. Its only drawback is that the `requirements.txt` remains on the image, but well, that's an image used for testing purposes, not for production and the `requirements.txt` file is pretty small, therefore it won't hurt anyone there.